target/linux/bcm63xx/base-files/etc/diag.sh

```diff
@@ -61,6 +61,9 @@ set_state() {
 	comtrend,vr-3026e)
 		status_led="VR-3026e:green:power"
 		;;
+	comtrend,vr-3032u)
+		status_led="VR-3032u:green:power"
```

Comment:
OT: This file looks like it would be relatively trivial to use generic diag.sh on this target. Do you think it would be worth it, or are you planning to replace with bmips soon anyway?

Reply:
I would like to switch to bmips target, but @KanjiMonster should also agree with this decission, and so far I haven't been able to contact him about it...